src/utils/string/StringUtils.js

```diff
@@ -64,9 +64,8 @@ const detectStateInCity = (city) => {
  * @returns {String}
  */
 const displayCityName = (city, state) => {
-  if (city === null || city === undefined || city === '') throw new Error('City must be defined');
-  if (state === null || state === undefined || state === '')
-    throw new Error('State must be defined');
+  if (city === null || city === undefined || city === '') return null;
+  if (state === null || state === undefined || state === '') return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Returning `null` when city or state is empty, null, or undefined could lead to unexpected behavior in other parts of the code that use this function. It would be better to return a default value or an empty string instead.
```diff
-  if (city === null || city === undefined || city === '') return null;
-  if (state === null || state === undefined || state === '') return null;
+  if (!city) return '';
+  if (!state) return city;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->